### DISCUSSION TITLE
Air Assist + Lights

### DISCUSSION THREAD

**Discussion by Matisak** [synman/Octoprint-Bettergrblsupport#123] [Q&A]
Is there any way to enable air assist(when enabled on Light burn) with this? Any way to add some macro or something to turn on lights?

**Comment by clevetbs** [comment]
i got myself a sonff flash tasmota and connected it to octoprint. set it to auto turn on air assist and exhaust fan.

**Comment by synman** [comment]
I know this is really old and my apologies for the neglect -- necromancy in progress.  

the short answer is yes!  -- as @clevetbs  alludes it does take a little bit of setup.  I personally use tp_link outlets but there are many many options out there.  

<img width="681" height="301" alt="Screenshot 2025-11-08 at 9 45 41 AM" src="https://github.com/user-attachments/assets/cd80b7a9-2534-4e69-be06-fd24a3362718" />
